san-francisco-tourism-solution/src/test/java/com/javaone/hol2017/junit5/CableCarsTest.java

```diff
@@ -1,15 +1,17 @@
 package com.javaone.hol2017.junit5;
 
-import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
```

Comment:
Can you revert these changes and make the code still work?
less is more in a Pull request 👍